### PR TITLE
STENCIL-2489 Upgrade Theme Editor in the CLI

### PR DESCRIPTION
**Pull request by mcampa**
Upgrade Theme Editor in the CLI

@bc-ong @mjschock @nickdengler

### CODE REVIEW

**Comment by mcampa** [comment]
All files changes under `server/plugins/stencil-editor/public/dist` were autogenerated by webpack.

ping @bc-ong @lord2800

File: package.json [hunk 1]
```diff
@@ -78,26 +78,26 @@
...
     "babel-preset-es2015": "^6.6.0",
-    "bcapp-pattern-lab": "git+ssh://git@github.com:bigcommerce-labs/bcapp-pattern-lab",
+    "bcapp-pattern-lab": "git@github.com:bigcommerce-labs/bcapp-pattern-lab.git",
     "clean-webpack-plugin": "^0.1.9",
     "compression": "^1.6.1",
     "copy-webpack-plugin": "^2.1.3",
     "es6-micro-loader": "^0.2.1",
     "express": "^4.13.4",
     "js-cookie": "^2.1.1",
-    "jschannel": "meenie/jschannel",
```
Comment (bc-ong) [inline package.json]: Lol.  We still had references to Cody's repo?

File: server/plugins/stencil-editor/stencil-editor.module.js [hunk 2]
```diff
@@ -132,15 +126,14 @@ module.exports.register = function (server, options, next) {
...
  */
-handlers.onRequest = function(request, reply) {
+handlers.onRequest = (request, reply) => {
     request.app.decorators = request.app.decorators || [];
 
     // Only add the SDK if stencilEditor is a query parameter or the cookie preview_config_id is set
     if (request.query.stencilEditor || (request.headers.cookie || '').indexOf('stencil_preview') !== -1) {
-
         request.app.decorators.push(content => {
             var scriptTags = `<script src="//localhost:${internals.options.stencilEditorPort}/dist/sdk.js"></script>\n`;
```
Comment (bc-ong) [inline server/plugins/stencil-editor/stencil-editor.module.js]: Update var to const?

Comment (bc-ong) [inline server/plugins/stencil-editor/stencil-editor.module.js]: Update var to const?

File: server/plugins/stencil-editor/stencil-editor.module.js [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+const handlers = {};
+const internals = {
+    options: {},
 };
-var themeConfig = ThemeConfig.getInstance();
+const themeConfig = ThemeConfig.getInstance();
 
-module.exports.register = function (server, options, next) {
+module.exports.register = (server, options, next) => {
     var configurationId;
```
Comment (bc-ong) [inline server/plugins/stencil-editor/stencil-editor.module.js]: Replace var with let on this line and the next?

Comment (mcampa) [inline server/plugins/stencil-editor/stencil-editor.module.js]: `let` is not supported in our current version of node without the `--harmony` flag

Comment (bc-ong) [inline server/plugins/stencil-editor/stencil-editor.module.js]: boo.

**Comment by bc-ong** [comment]
👍

**Pull request merged**